server/app/util/constants.py

```diff
@@ -28,5 +28,5 @@ class DiffExpMode(AugmentedEnum):
 
 
 JSON_NaN_to_num_warning_msg = (
-    "JSON encoding failure - suggest trying --nan-to-num command line option"
+    "JSON encoding failure - please verify all data are finite values (no NaN or Infinities)"
```

Comment:
Could we add a comment here that when working with datasets including non-finite values to use flatbuffer routes instead of JSON? Low priority, I'm mainly thinking about external developers here.

Reply:
well, in theory this error will never occur again, as the front-end doesn't call those routes (and nobody else uses the back-end).    I simply changed this error as prelude to removing the whole set of unused JSON routes...

Reply:
yeah, that's fine

Reply:
cleanup reminder filed as issue #550